packages/core/src/tenants/SystemContext.ts

```diff
@@ -34,6 +37,13 @@ export default class SystemContext {
           hostnameProviderDataGuard
         );
       })(),
+      (async () => {
```

Comment:
Wondering,  whether we should block the user flow if any of the provided listed here throws an error.

Reply:
We will block the whole Logto app from start if the config is not empty and has errors.